src/app/components/harvest/components/metadata-review/file-row.component.ts

```diff
@@ -84,13 +93,19 @@ interface ValidationMessage {
         ></fa-icon>
       </div>
     </div>
+    <ng-container *if="updateChevronState(validationsContainer)"></ng-container>
```

Comment:
This is a hacky workaround I had to implement because change detection wasn't firing for `*ngFor` changes

Reply:
This has now been resolved. It seems to have been an issue with the logic in my `updateChevronState` method